tlspuffin/src/tls/key_exchange.rs

```diff
@@ -70,7 +70,7 @@ pub fn tls12_new_secrets(
         .try_into()
         .map_err(|_| FnError::Unknown("Server random did not have length of 32".to_string()))?;
     let randoms = ConnectionRandoms {
-        client: [1; 32], // todo https://github.com/tlspuffin/tlspuffin/issues/129
+        client: client_random.0,
         server: server_random,
     };
     let kx = tls12_key_exchange(group)?;
```

Comment:
Is there other tls12 possibilities?

Reply:
I opened the issue #368 to investigate encryption/decryption of TLS 1.2 message